src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/SaveOidcngResourceServerEntityCommand.php

```diff
@@ -143,18 +144,23 @@ class SaveOidcngResourceServerEntityCommand implements SaveEntityCommandInterfac
      */
     private $manageId;
 
+    /** @var bool */
+    private $isCopy;
+
     public function __construct()
     {
     }
```

Comment:
:man_shrugging: If you feel like a good boy scout today. Maybe remove this constructor while you are in this file :evergreen_tree: